run_nomad.sh

```diff
@@ -17,7 +17,6 @@ if [ ! -d $nomad_dir ]; then
 fi
 
 # Start the nomad in development locally
-# TODO: do this in a Nomad container.
```

Comment:
Wait I didn't mean just remove this TODO, but rather to have this actually invoke `run_nomad_docker.sh`.

Reply:
I thought about that, but I think having Nomad Dockerized is more of a convenience for development and testing, and that there may be situations where a dev or a staging server would want to run a local, non-Dockerized instance of Nomad.